Reference/Routing/custom-controllers.md

```diff
@@ -60,7 +60,17 @@ If you are returning a custom model, then this directive will need to change bec
 Please note that if your template uses a layout that expects the model to be of type `Umbraco.Web.Models.RenderModel` then changing the template to inherit from `Umbraco.Web.Mvc.UmbracoViewPage<MyCustomModel>` will cause an exception. This is due to the way ASP.NET MVC works with strongly typed views: the requirement for a specific type applies all the way from the top-most layout down to the template. There are two ways to solve this problem:
 
 1. Break the dependency on `Umbraco.Web.Models.RenderModel` in your layout by having it inherit from `Umbraco.Web.Mvc.UmbracoViewPage<dynamic>` (this means `@Model` will be of type `dynamic` in the layout).
-2. Make your custom model inherit from `Umbraco.Web.Models.RenderModel` (this means `@Model...` will continue to work in the layouts used by your template).
+2. Make your custom model inherit from `Umbraco.Web.Models.RenderModel` and ensure you pass through the Umbraco model thus:-
+
+  public class MyNewViewModel : RenderModel
+  {
+      //Standard Model Pass Through
+      public MyNewViewModel(IPublishedContent content) : base(content) { }
+
+      //Custom properties here…..
+      public string myProperty1 { get; set; }
+      public string myProperty2 { get; set; }
+  } (this means `@Model...` will continue to work in the layouts used by your template).
 
```

Comment:
This class declaration needs to be indented for code formatting to work and the last sentence needs some line breaks after the code block. Also please use the correct c# naming conventions for your properties, these should be MyProperty1 and MyProperty2